compiler/semantic/expr.go

```diff
@@ -244,6 +244,20 @@ func (a *analyzer) semExpr(e ast.Expr) dag.Expr {
 			Kind:    "MapExpr",
 			Entries: entries,
 		}
+	case *ast.TupleExpr:
+		elems := make([]dag.RecordElem, 0, len(e.Elems))
+		for colno, elem := range e.Elems {
+			e := a.semExpr(elem)
+			elems = append(elems, &dag.Field{
+				Kind:  "Field",
+				Name:  fmt.Sprintf("c%d", colno),
```

Comment:
Isn't there a concern here that this could collide with pre-existing column names?

Reply:
The tuple type doesn't have column names so this is assigning `c0,c1,...` as the column names of the value that comprises the tuple.  No possibility of field name conflict.